fl4health/clients/ditto_client.py

```diff
@@ -177,40 +184,27 @@ def initialize_all_model_weights(self, parameters: NDArrays, config: Config) ->
         self.parameter_exchanger.pull_parameters(parameters, self.model, config)
         self.parameter_exchanger.pull_parameters(parameters, self.global_model, config)
 
-    def train_by_epochs(
-        self, epochs: int, current_round: Optional[int] = None
-    ) -> Tuple[Dict[str, float], Dict[str, Scalar]]:
-        """
-        Train locally for the specified number of epochs.
-
-        Args:
-            epochs (int): The number of epochs for local training.
-            current_round (Optional[int]): The current FL round.
-
-        Returns:
-            Tuple[Dict[str, float], Dict[str, Scalar]]: The loss and metrics dictionary from the local training.
-                Loss is a dictionary of one or more losses that represent the different components of the loss.
-        """
-        # Need to also set the global model to train mode
-        self.global_model.train()
-        return super().train_by_epochs(epochs, current_round)
+    def save_initial_global_tensors(self) -> None:
```

Comment:
nit: Maybe set_initial_global_tensors is a more fitting name?

Reply:
Sure!